components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -165,15 +165,19 @@ export const buildModel = ({
 		latestItemsTime,
 		latestItemsAgeHours
 	})
-
 	if (itemGroups.length > 0 && customSlotContent) {
-		const insertPosition = Math.min(customSlotPosition, items.length)
-		const insert = getGroupAndIndex(itemGroups, insertPosition)
-		const copyOfItems = [...itemGroups[insert.group].items]
+		customSlotContent = Array.isArray(customSlotContent) ? customSlotContent : [customSlotContent]
```

Comment:
This loop could be rewritten as
```suggestion
		for (const slotPosition of customSlotPositions) { ... }
```
which would give direct access to the values of the array of slot positions.